internal/plugin/validators/lua_validator_test.go

```diff
@@ -260,7 +260,7 @@ func TestProcessAutoFields(t *testing.T) {
 		}
 		require.Equal(t, expectedConfig, processConfig)
 	})
-	t.Run("fails to process defaults with plugin schema which has not been loaded", func(t *testing.T) {
+	t.Run("processing defaults with plugin schema which has not been loaded return an empty config", func(t *testing.T) {
```

Comment:
this test was previously working because goks was returning invalid json. `ProcessDefaults` never returns error so I fixed it to checked that the plugin config is nil.